### ISSUE TITLE
Delete messages

### ISSUE THREAD

**Issue by SamarthKadam** [SamarthKadam/ChatBox#278]
When you hover over the message end there should be button that says delte messages. 

Implementation: When the user clicks delelte message. You don't need to delte that message. Just add a message saying this message has been deleted with specific design like below
![Screenshot (609)](https://github.com/SamarthKadam/ChatBox/assets/87573175/1ececa06-62b7-4a4b-9bb1-59811ad85309)

**Comment by Ar7109** [comment]
@SamarthKadam Please assign this issue to me under gssoc 2024

**Comment by SamarthKadam** [comment]
@Ar7109 Assigned

**Comment by SamarthKadam** [comment]
@Ar7109 Are you working on it?

**Comment by Ar7109** [comment]
@SamarthKadam Yes I am working upon the issue and I will try to do that as soon as possible

**Comment by SamarthKadam** [comment]
@Ar7109 as per implementation just add field in message schema isDeleted. if deleted then display content saying Message is deleted. Im increasing the level as it involves working in backend and frontend as well. So Good Luck.

**Comment by Ar7109** [comment]
@SamarthKadam Thanks a lot I will try to resolve it as soon as possible

**Comment by abhigyanverma111** [comment]
please assign me this issue under gssoc 2024

**Comment by SamarthKadam** [comment]
@abhigyanverma111 You can start working on this issue If your able to come up with this feature before @Ar7109 it'll be merged.

**Comment by SamarthKadam** [comment]
@abhigyanverma111 @Ar7109 Any status on this?
